docs/providers/aws/guide/functions.md

```diff
@@ -37,6 +37,7 @@ functions:
     runtime: python2.7 # optional overwrite, default is provider runtime
     memorySize: 512 # optional, in MB, default is 1024
     timeout: 10 # optional, in seconds, default is 6
+    reservedConcurrency: 5 # optional, reserved concurrency limit for this function. By default, it is not set and AWS use Account concurrency limit
```

Comment:
Very minor suggestion on grammar:
> By default, it is not set and AWS use Account concurrency limit

'use' should be 'uses', account doesn't need capitalization.

> By default, it is not set and AWS uses account concurrency limit

Reply:
I fixed this sentence.